lib/zeromq/majordomo/asynchronous_arbiter.rb

```diff
@@ -0,0 +1,49 @@
+require 'ffi-rzmq'
+
+module Zeromq
+  module Majordomo
+    class AsynchronousArbiter
+      INFINITE = -1
+
+      class << self
```

Comment:
Got it. I was stuck on the listener stuff but that's not necessary for this async arbiter. Let me rework it real quick.